### PR TITLE
[github] Add default repository support

### LINKED ISSUE #140: Allow GitHub repo name to be stored as a config parameter.
**Is your feature request related to a problem? Please describe.**

The GitHub commands require specifying a repo name. This will get repetitive and annoying.
**Describe the solution you'd like**
Allow a default repo to be specified and make the repo name otherwise optional. Commander secrets could be used to store the configuration of the default repo to use.

### PR DESCRIPTION
**Pull request by satyarohith**
Also small improvements to codebase and output of commands. 

### `github_close_issue`
Before:
close_issue
After:
close_issue

### `github_create_issue`
Before:
create_issue
After:
create_issue

### `github_label`
Before:
label_issue
After:
label_issue

### `github_find_pr` ~(No output changes)~
Before:
find_pr
After:
find_pr

### `github_stats`
Before:
github_stats
After:
github_stats

### `github_view_pr` ~(No output changes)~
Before:
github_stats
After:
github_stats

### `github_reopen_issue`
Before:
reopen_issue.
After:
reopen_issue.

### `github_request_review`
Before:
request_review
After:
request_review

Closes #140

### CODE REVIEW

**Review by rabbah — COMMENTED** [review]
❤️ the improvements.

File: github/packages/github/github_create_issue.js [hunk 1]
```diff
@@ -1,61 +1,64 @@
...
-    
-  if (!secrets.github_token) {
+  const {github_token: githubToken, github_default_repo: defaultRepo} = secrets;
+  if (!githubToken) {
     return {
-      response_type: 'in_channel',
-      text: 'Missing GitHub Personal Access Token!'
+      response_type: 'ephemeral',
+      text:
+        'Missing GitHub Personal Access Token! Create a secret named `github_token` with your personal access token. '
```
Comment (rabbah) [inline github/packages/github/github_create_issue.js]:
```suggestion
        'Missing GitHub Personal Access Token! Create a secret named `github_token` with your personal access token.'

```

**Comment by satyarohith** [comment]
Sorry for the big PR. I'm happy to get on a call for review if it helps.

File: github/packages/github/github_create_issue.js [hunk 2]
```diff
@@ -1,61 +1,64 @@
...
+  const result = [];
+  const {title, body} = params;
+  const repo = params.repo === false ? defaultRepo : params.repo;
+  if (!repo && !defaultRepo) {
     return {
-      response_type: 'in_channel',
-      text: data.response.headers.status
+      response_type: 'ephemeral',
+      text:
+        'Please create `github_default_repo` secret to avoid passing the repository.'
```
Comment (rabbah) [inline github/packages/github/github_create_issue.js]: I suggest:

Either pass a repo name or create a secret ...

**Comment by satyarohith** [comment]
Oops! I need to update the docs.

**Review by rabbah — APPROVED** [review]
Thanks for these enhancements.

LGTM.

**Comment by satyarohith** [comment]
Please rebase and commit instead of squash.

**Comment by niks3089** [comment]
Changes LGTM. Conflict file needs to be resolved

**Comment by satyarohith** [comment]
@niks3089 Resolved the conflicts.

**Pull request merged**